pkg/testrunner/result/ingest.go

```diff
@@ -35,7 +35,7 @@ const (
 func IngestDir(path string, esCfgName string) error {
 	files, err := ioutil.ReadDir(path)
 	if err != nil {
-		return fmt.Errorf("Cannot read directory %s: %s", path, err.Error())
+		return fmt.Errorf("cannot read directory '%s'd: %s", path, err.Error())
```

Comment:
nit: Cannot/cannot (gland displays and error but i think we should be consistent in our error messages)

Reply:
Yes, I know that we use upper-case so far, but I'd follow https://github.com/golang/go/wiki/CodeReviewComments#error-strings
Either we can change them over time as we change nearby code (this is what I did), or change them in one shot everywhere, or leave them upper-case (which I think would be the least good option).
WDYT?

Reply:
your're right. We should change them over time.
Maybe we can include this when doing the error refactoring